yarl/_parse.py

```diff
@@ -69,11 +69,11 @@ def split_url(url: str) -> SplitURLType:
             # Valid bracketed hosts are defined in
             # https://www.rfc-editor.org/rfc/rfc3986#page-49
             # https://url.spec.whatwg.org/
-            if bracketed_host[0] == "v":
+            if bracketed_host.startswith("v"):
```

Comment:
```suggestion
            if bracketed_host and bracketed_host[0] == "v":
```

This is faster since there is no function call